src/types/formatted/vetCenterLocationListing.ts

```diff
@@ -1,11 +1,70 @@
-import { VetCenter } from './vetCenter'
 import { PublishedEntity } from './publishedEntity'
+import {
+  FieldAddress,
+  FieldGeoLocation,
+  FieldOfficeHours,
+} from '../drupal/field_type'
+import { FacilityOperatingStatusFlags } from '../drupal/node'
+import { MediaImage as FormattedMediaImage } from './media'
+
+/**
+ * Common fields shared across all Vet Center types (both full and listing versions).
+ */
+export type CommonVetCenterFields = {
+  title: string
+  path: string
+  // administration: FieldAdministration
+  address: FieldAddress
+  geolocation: FieldGeoLocation
+  lastSavedByAnEditor: string | null
+  image: FormattedMediaImage
+  fieldFacilityLocatorApiId: string
+}
+
+/**
+ * Main Vet Center info type for location listings.
+ * This is a simplified version of VetCenter for listing purposes.
+ */
+export type VetCenterLocationInfo = CommonVetCenterFields & {
```

Comment:
Like the formatter function, this data will also likely change as I figure out what's truly necessary.